src/DistributedMetadata/CatalogService.cpp

```diff
@@ -695,10 +704,11 @@ void CatalogService::mergeCatalog(const NodePtr & node, TableContainerPerNode sn
         }
 
         {
-            /// FIXME, if table definition changed, we will need update the storage inline
```

Comment:
We didn't fix this `FIXME` in this PR. For example, table column `i` get changed from `UInt32` to `Int32` in which case, we will need update the `storage` object in `storages` map.

I am good to leave this FIXME open and get this PR merged for now.

Reply:
fixed